kiwi/builder/disk.py

```diff
@@ -435,7 +435,11 @@ def create_disk(self):  # noqa: C901
         if self.root_filesystem_is_overlay:
             squashed_root_file = NamedTemporaryFile()
             squashed_root = FileSystemSquashFs(
-                device_provider=None, root_dir=self.root_dir
+                device_provider=None, root_dir=self.root_dir,
+                custom_args={
+                    'compression': self.xml_state.
+                    build_type.get_squashfscompression()
+                }
```

Comment:
For consistency and because used at other places in the code, can we change the line break to match the following style:

```python
custom_args={
    'compression':
        self.xml_state.build_type.get_squashfscompression()
```

Reply:
:+1: